Macros/Fmt_occ_2018f.sas

```diff
@@ -0,0 +1,563 @@
+      8720 = "Extruding, forming, pressing, and compacting machine setters, operators, and tenders"
+      8730 = "Furnace, kiln, oven, drier, and kettle operators and tenders"
+      8740 = "Inspectors, testers, sorters, samplers, and weighers"
+      8750 = "Jewelers and precious stone and metal workers"
+      8760 = "Dental and ophthalmic laboratory technicians and medical appliance technicians"
+      8800 = "Packaging and filling machine operators and tenders"
+      8810 = "Painting workers"
+      8830 = "Photographic process workers and processing machine operators"
+      8850 = "Adhesive bonding machine operators and tenders"
+      8910 = "Etchers and engravers"
+      8920 = "Molders, shapers, and casters, except metal and plastic"
+      8930 = "Paper goods machine setters, operators, and tenders"
+      8940 = "Tire builders"
+      8950 = "Helpers--production workers"
+      8990 = "Miscellaneous production workers, including equipment operators and tenders"
+      9005 = "Supervisors of transportation and material moving workers"
+      9030 = "Aircraft pilots and flight engineers"
+      9040 = "Air traffic controllers and airfield operations specialists"
+      9050 = "Flight attendants"
+      9110 = "Ambulance drivers and attendants, except emergency medical technicians"
+      9121 = "Bus drivers, school"
+      9122 = "Bus drivers, transit and intercity"
+      9130 = "Driver/sales workers and truck drivers"
+      9141 = "Shuttle drivers and chauffeurs"
+      9142 = "Taxi drivers"
+      9150 = "Motor vehicle operators, all other"
+      9210 = "Locomotive engineers and operators"
+      9240 = "Railroad conductors and yardmasters"
+      9265 = "Other rail transportation workers"
+      9300 = "Sailors and marine oilers, and ship engineers"
+      9310 = "Ship and boat captains and operators"
+      9350 = "Parking attendants"
+      9365 = "Transportation service attendants"
+      9410 = "Transportation inspectors"
+      9415 = "Passenger attendants"
+      9430 = "Other transportation workers"
+      9510 = "Crane and tower operators"
+      9570 = "Conveyor, dredge, and hoist and winch operators"
+      9600 = "Industrial truck and tractor operators"
+      9610 = "Cleaners of vehicles and equipment"
+      9620 = "Laborers and freight, stock, and material movers, hand"
+      9630 = "Machine feeders and offbearers"
+      9640 = "Packers and packagers, hand"
+      9645 = "Stockers and order fillers"
+      9650 = "Pumping station operators"
+      9720 = "Refuse and recyclable material collectors"
+      9760 = "Other material moving workers"
+      9800 = "Military officer special and tactical operations leaders"
+      9810 = "First-line enlisted military supervisors"
+      9825 = "Military enlisted tactical operations and air/weapons specialists and crew members"
+      9830 = "Military, rank not specified"
+      9920 = "Unemployed, with no work experience in the last 5 years or earlier or never worked"
+
```

Comment:
Missing semicolon here is preventing this from running automatically from the main program.